test/pre-test-scripts.ts

```diff
@@ -1,48 +1,26 @@
 import dotenv from 'dotenv';
 import * as path from 'path';
-import { initServer } from '../src/server';
+import { dbConnection, initDbConnection, initServer } from '../src/server';
 import { AppDataSource } from '../src/dataSource';
-import { assert } from 'chai';
 import { SiweNonce } from '../src/entities/siweNonce';
 
 dotenv.config({
   path: path.resolve(__dirname, `../config/${process.env.NODE_ENV || ''}.env`),
 });
-/* eslint-disable @typescript-eslint/no-var-requires */
-const { dropdb, createdb } = require('pgtools');
 
 async function dropDatabaseAndCreateFreshOne() {
-  const config = {
-    user: process.env.TYPEORM_DATABASE_USER,
-    password: process.env.TYPEORM_DATABASE_PASSWORD,
-    port: process.env.TYPEORM_DATABASE_PORT,
-    host: process.env.TYPEORM_DATABASE_HOST,
-  };
-
   // tslint:disable-next-line:no-console
   console.log('Dropping DB');
   try {
-    await dropdb(config, process.env.TYPEORM_DATABASE_NAME);
+    await AppDataSource.synchronize(true);
```

Comment:
I think we should keep the config of dropdb, and createdb if posible. 
Maybe we have to reconfigure the app or someone new joins, so its an easy setup to just run tests and all works, instead of manually.
But keep it optional maybe configurable with an env.

What do you think?